x11-terms/kitty/kitty-9999.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Comment:
Do any released versions in tree support 3.9?

Reply:
Yes, the ebuild for the 0.20.3 version of kitty supports python 3.9